lib/reverse_sentence.rb

```diff
@@ -1,6 +1,56 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: ? O(n)
+# Space complexity: ? O(1)
```

Comment:
Make sure to explain your reasoning behind your choices for Big O time and space complexity.